public/assets/js/forms.js

```diff
@@ -659,21 +662,21 @@ $(function(){
 
         // Video
         var _addVideo = function(e) {
-            var val = $(this).val();
+            let val = $(this).val();
             if(!val) return;
-            var video = parseVideoURL(val);
-            var input = this;
-            var $container = $(this).closest('.media-container');
-            var $holder = $container.find('.video-holder');
-            var $embed = $container.find('.embed-responsive');
+            let video = parseVideoURL(val);
+            let input = this;
+            let $container = $(this).closest('.media-container');
+            let $holder = $container.find('.video-holder');
+            let $embed = $container.find('.embed-responsive');
```

Comment:
Why create this fields as let instead of const? Following JS developers I see that they tend to use const in this cases instead of let. We could try to define a common ground for this JS developments. @carlos-mg89

Reply:
I'd need to review deeper that block and see which fields should be dynamic (let) and which ones are not (const), but you're right @davidbeig 

To be honest, I simply updated the code from var to let without thinking very much about this. We should definitely sit down and talk about having a convention with this.

If you want, you could start a document to have the common JS (perhaps another for PHP too) and we could talk about it in the next Goteo meeting.

Reply:
I sure will!